### ISSUE TITLE
BigEgg's 1.0.6.1 Issue Report Thread

### ISSUE THREAD

**Issue by BigEgg17** [EpochModTeam/DayZ-Epoch#1915]
It appears the option to press shift-minus and enter commands (usually people use endmission to dupe) is prevented by default, however, it can still be accomplished by simply opening your map or pressing escape (I am sure there are other things you can open to bypass the default preventative measure) and doing it while you have the dialogs open.

**Comment by icomrade** [comment]
I made sure to cover these displays by default in 1.0.6, are you sure infistar or something is overwriting the display EH for these Displays?

**Comment by BigEgg17** [comment]
Nothing is running on my test server besides Ebay's test kit and single currency with a bunch of other little things (none that override display EH's). I mean I guess it could be the test kit, I will try removing it, but I am doubtful.

**Comment by BigEgg17** [comment]
Another Issue:

When building, a player can right click to build the item (for example a safe), open their gear and drop the safe on the ground, then press space-bar to build the item, duping the safe.

I fixed this by simply adding:

`DZE_cancelBuilding = true;`

to the gear statement in keyboard.sqf

**Comment by ebayShopper** [comment]
I removed the keydown EH from the map in 416fdbc, but I just tested again and I was wrong. It appears it is still needed. Readded with 665ecd9. I can not enter cheats on the abort menu or any of its sub menus @BigEgg17. Can you clarify which display exactly you are able to enter cheats on? A screenshot would help.

I just tested the build dupe you mentioned and it does not work for me with the default Epoch settings. My player does the build animation, but at the end it fails to build and says canceled building. The safe is on the ground but there is none in my inventory.

**Comment by BigEgg17** [comment]
Apologies @ebayShopper, I negated to try the escape menu, as when I saw it worked with the map I assumed it worked with the escape menu as well, unaware that it had been patched. That bug seems to be solved with your commit.

As for the build dupe, I am also getting the same thing that you mentioned, cancelling building because it cannot find the safe in the inventory. I assumed that since it would let me place it, the bug still existed as in 1.0.5.1. That bug seems to be non-existent - sorry for that.

As for other problems, I will be testing approx. two dupe methods that players are reporting to me and including information as to what they are in this issue report if that is okay (idk how long these remain open).

Thank you for the map fix! :)
